system/HTTP/MessageInterface.php

```diff
@@ -0,0 +1,112 @@
+<?php
+
+/**
+ * This file is part of the CodeIgniter 4 framework.
+ *
+ * (c) CodeIgniter Foundation <admin@codeigniter.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace CodeIgniter\HTTP;
+
+use CodeIgniter\HTTP\Exceptions\HTTPException;
+
```

Comment:
```suggestion
```
Do we need this here?

Reply:
It is necessary for the static analysis to recognize the type on `prependHeader()`. I could fully-qualify it there but I usually header them even for SA.